src/bijectors/corr.jl

```diff
@@ -173,6 +173,23 @@ end
 
 inverse(::typeof(vec_to_triu1)) = triu1_to_vec
 
+function vec_to_triu1_row_index(idx)
+    # Assumes that vector was saved in a column-major order
+    # and that vector is one-based indexed.
+    M = _triu1_dim_from_length(idx - 1)
+    return idx - (M*(M-1) ÷ 2)
+end
+
+abstract type AbstractVecCorrBijector <: Bijector end
```

Comment:
With this approach, `bijector` becomes type-unstable (see comment on this further below).

We should probably just make `VecCorrBijector` hold the `uplo` of the `Cholesky`, and add an if-statement to the `transform`, etc. This way we can avoid type-instabilities.